api/util/logging-helpers.js

```diff
@@ -0,0 +1,41 @@
+const { LOGGING_API_URL } = require('../config/config');
+const axios = require('axios');
+
+/**
+ * Add a new error by calling on the ErrorLog API.
+ * @param {Object} newError - The error that is to be added
```

Comment:
Is this a parameter here?

Reply:
doesn't seem to be to used in this function

Reply:
uh oh!!!! got me